pkgs/applications/editors/molsketch/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, mkDerivation, fetchgit, cmake, pkg-config, hicolor-icon-theme, openbabel, desktop-file-utils, qttranslations }:
+
+mkDerivation {
+  pname = "molsketch";
+  version = "unstable-2022-09-18";
```

Comment:
Is there a particular reason, why you want to use an unstable version? The latest stable is just 3 months old.

Reply:
I believe this is the same revision as the latest stable (?).  I didn't want to use the tarballs and instead directly get the source from git, but upstream didn't have a tag for the stable release.

Reply:
There's nothing wrong with tar balls for that purpose. I would recommend here to pick the stable version from sourceforge with a mirror URL (i.e. ` url = "mirror://sourceforge/...`).